### ISSUE TITLE
ANC Import: uses wrong external_id for the SCA number

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2479]
I used the collected material external id which also makes the SCA number an identifier (but it is not unique!).
I should use the small find external id instead.

**Comment by lbestock** [comment]
It is absent now. There is a field still for SCA# in CM, but it is blank (looking at one we know was registered to the SCA - 37002). There is no field for SCA# (by any name) in Small Finds.

**Comment by lbestock** [comment]
Whoa. That should just have been a configuration issue. So I went to configuration and changed visibility so that there was no visibility for CM alternate number, and small finds alternate number was labeled SCA. And then I went to search for 37002 again and now that identifier is unknown?

**Comment by lbestock** [comment]
@urapadmin this is weird beyond belief. I am in recording. If I go to the start page and search 37002 I get nothing. (That is how I found the record prior to changing configuration.) And if I go to unit 172 and look at CM by unit, it is not in the long list of CM I can find there. But if I go to 172 locus 13 and look at the CM for the locus, there 37002 is. With its SCA# in the proper place on the small finds tab.

**Comment by lbestock** [comment]
37002 shows up in the navigation of recent places.

**Comment by lbestock** [comment]
37004, 37005, same thing...

**Comment by urapadmin** [comment]
> @urapadmin this is weird beyond belief. I am in recording. If I go to the start page and search 37002 I get nothing. (That is how I found the record prior to changing configuration.) And if I go to unit 172 and look at CM by unit, it is not in the long list of CM I can find there. But if I go to 172 locus 13 and look at the CM for the locus, there 37002 is. With its SCA# in the proper place on the small finds tab.

I suspect that is just a not update file identifier cache after my import. As soon as the server restarts or there is a synchronization run this will automatically be mended. It affects only the search,

**Comment by lbestock** [comment]
In recording? And it affects the list in CM by unit, not just search.

**Comment by urapadmin** [comment]
we established that this is an issue with the sorting order of the CM by unit list. The missong collected materials were there but not where you expected them to be in the list.

**Comment by urapadmin** [comment]
so this here should be okay by now, no?

**Comment by urapadmin** [comment]
`So I went to configuration and changed visibility so that there was no visibility for CM alternate number, and small finds alternate number was labeled SCA. And then I went to search for 37002 again and now that identifier is unknown?`

I can confirm that. When changing stuff in the configuration I reset some information and that can affect the whole search. That is okay and we can live with it. It heals automatically after the next synchronization.
